### ISSUE TITLE
NTR: Aspartylglucosamine high in blood (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8524]
New term request Aspartylglucosamine high in blood (segal_030506142657): 2-acetamido-1-L-beta-aspartamido-1,2-dideoxy-beta-D-glucose

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000282
- label: Elevated circulating aspartylglucosamine concentration
- parents: [Abnormal circulating non-proteinogenic amino acid concentration (HP:0033109)](https://hpo.jax.org/app/browse/term/HP:0033109)
- pmid: 
- def: Concentration of aspartylglucosamine in the blood circulation above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8524
- synonyms: Aspartylglucosamine high in blood(NONE)
